test/test_pipeline/implementations/test_CategoryShift.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import numpy as np
+import scipy.sparse
+
+from autosklearn.pipeline.implementations.CategoryShift import CategoryShift
+
+
+class CategoryShiftTest(unittest.TestCase):
+
+    def test_dense(self):
+        X = np.random.randint(0, 255, (3, 4))
+        Y = CategoryShift().fit_transform(X)
+        self.assertTrue((Y == X + 3).all())
+
+    def test_sparse(self):
+        X = scipy.sparse.csc_matrix(
+            ([1, 2, 0, 4], ([0, 1, 2, 1], [3, 2, 1, 0])), shape=(3, 4))
+        Y = CategoryShift().fit_transform(X)
+        X.data += 3
+        self.assertTrue((Y.todense() == X.todense()).all())
```

Comment:
Done. To make it uniform with the other classes, I added this test at the component level (not at the implementation). I think it also makes more sense to do it there, because this is the class that has the dictionary showing the datatypes.